.github/workflows/ci.yml

```diff
@@ -42,3 +42,7 @@ jobs:
 
       - name: Run tests
         run: bundle exec rake
+
+      - name: Run index troubleshooting tool
+        if: matrix.os != 'windows-latest'
```

Comment:
I could not figure how why, but on Windows this fails with `bundler: not executable`.